apps/api/src/app/integrations/integrations.controller.ts

```diff
@@ -195,4 +198,14 @@ export class IntegrationsController {
 
     return result;
   }
+
+  @Get('/in-app/activated')
```

Comment:
thinking out loud:
```suggestion
  @Get('/in-app/status')
```

What do you think about this one?

Reply:
@scopsy I like it but then would a boolean response will make sense in your opinion, or should we update the response to an object?

Reply:
We can get an object which we can later add other metada to it

Reply:
Tent to agree :)